com.woltlab.wcf/templates/articleListItems.tpl

```diff
@@ -12,77 +12,81 @@
 
 			{unsafe:$view->renderInteractionContextMenuButton($article)}
 		</div>
-		
-		<div class="contentItemLink">
-			<div class="contentItemImage contentItemImageLarge">
-				<img
-					class="contentItemImageElement"
-					src="{if $article->getTeaserImage()}{$article->getTeaserImage()->getThumbnailLink('medium')}{else}{$__wcf->getStyleHandler()->getStyle()->getCoverPhotoURL()}{/if}"
-					height="{if $article->getTeaserImage()}{@$article->getTeaserImage()->getThumbnailHeight('medium')}{else}{@$__wcf->getStyleHandler()->getStyle()->getCoverPhotoHeight()}{/if}"
-					width="{if $article->getTeaserImage()}{@$article->getTeaserImage()->getThumbnailWidth('medium')}{else}{@$__wcf->getStyleHandler()->getStyle()->getCoverPhotoWidth()}{/if}"
-					loading="lazy"
-					alt="">
-				
-				{hascontent}
-					<div class="contentItemBadges">
-						{content}
-							{if $article->isDeleted}<span class="badge label red contentItemBadge contentItemBadgeIsDeleted">{lang}wcf.message.status.deleted{/lang}</span>{/if}
-							{if !$article->isPublished()}<span class="badge label green contentItemBadge contentItemBadgeIsDisabled">{lang}wcf.message.status.disabled{/lang}</span>{/if}
-							{if $article->isNew()}<span class="badge label contentItemBadge contentItemBadgeNew">{lang}wcf.message.new{/lang}</span>{/if}
-							
-							{event name='contentItemBadges'}
-						{/content}
-					</div>
-				{/hascontent}
-			</div>
-			
-			<div class="contentItemContent">
-				{if $article->hasLabels()}
-					<div class="contentItemLabels">
-						{foreach from=$article->getLabels() item=label}
-							{@$label->render('contentItemLabel')}
-						{/foreach}
-					</div>
-				{/if}
-				
-				<h2 class="contentItemTitle"><a href="{$article->getLink()}" class="contentItemTitleLink">{$article->getTitle()}</a></h2>
-				
-				<div class="contentItemDescription">
-					{@$article->getFormattedTeaser()}
+
+		<div class="entryCardList__item__image">
+			<img
+				class="entryCardList__item__image__element"
+				src="{if $article->getTeaserImage()}{$article->getTeaserImage()->getThumbnailLink('medium')}{else}{$__wcf->getStyleHandler()->getStyle()->getCoverPhotoURL()}{/if}"
+				height="{if $article->getTeaserImage()}{$article->getTeaserImage()->getThumbnailHeight('medium')}{else}{$__wcf->getStyleHandler()->getStyle()->getCoverPhotoHeight()}{/if}"
+				width="{if $article->getTeaserImage()}{$article->getTeaserImage()->getThumbnailWidth('medium')}{else}{$__wcf->getStyleHandler()->getStyle()->getCoverPhotoWidth()}{/if}"
```

Comment:
Can we somehow add a helper method or an extra parameter that will implicitly fall back to the style’s cover photo?